src/Datadog.Trace.BenchmarkDotNet/Datadog.Trace.BenchmarkDotNet.csproj

```diff
@@ -14,7 +14,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" Version="0.12.1" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.13.0" />
```

Comment:
@tonyredondo our BenchmarkDotNet exporter is used at compile-time right? So upgrading to a newer version doesn't really limit our support?

Reply:
That’s why I asked to put this on hold 😊. In previous upgrades I had face it some exporter interfaces changes. So I have to evaluate if I can do this integration using reverse ducktyping and try to avoid the dependency. Or use a base version as the Msbuild integration.